server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -194,8 +194,13 @@ public enum MetadataVersion {
     // Add JBOD support for KRaft.
     IBP_3_7_IV2(17, "3.7", "IV2", true),
 
+    // IBP_3_7_IV3 was ELR related supports (KIP-966) and has been moved forward
```

Comment:
I think we should keep the value and just not have anything associated with it.  Call it "reserved", similar to IBP_3_7_IV1.  WDYT?

Reply:
I think that makes sense.

Reply:
Done!

Reply:
very small nit here -- but the comment could be a little clearer. Something like

IBP_3_7_IV3 was reserved for ELR changes (KIP-966) but has been deferred to a later MV

Reply:
Okay, I updated the comment to try to make it clearer. This is where we really need the KIP.